golang/labs/lab4/lab4.go

```diff
@@ -0,0 +1,47 @@
+package labs
+
+import (
+	"fmt"
+	"math"
+)
+
+// Лабораторная 4, задание по функции 6
+func calculate_y(x float64) float64 {
+	if math.Abs(x) >= 1 {
+		return math.Pow(1.2, x) - math.Pow(x, 1.2)
+	} else {
+		return math.Acos(x)
+	}
+}
+
+// Задача под А
+func Task_A(begin_x, end_x, delta_x float64) []float64 {
+	answer_arr := make([]float64, 0)
```

Comment:
```suggestion
	var answer_arr []float64
```

так лучше объявлять пустой массив

Reply:
Изменил